Source/Core/DiscIO/Enums.h

```diff
@@ -15,6 +15,7 @@ namespace DiscIO
 enum class Platform
 {
   GameCubeDisc = 0,
+  Triforce,
```

Comment:
Is there a reason why this is just `Triforce` instead of `TriforceDisc` (to match `GameCubeDisc` and `WiiDisc`)?

Reply:
Maybe because TriForce games can be discs or cartridges?

Reply:
And those two formats are identical data/format-wise?

Reply:
I'm not really sure, given nobody has extracted a correct image from the NAND dumps.

Reply:
But yes, the extracted images should be identical in theory

Reply:
I think the specifier for the Gamecube doesn't hurt, but for Triforce is different

Reply:
I mean the thing is the images dolphin is using arent the original disc dumps, theyve been extracted and decrypted from GD-ROMs to gamecube images, and the same thing would happen with the nand carts.